### ISSUE TITLE
feature request:  exit codes

### ISSUE THREAD

**Issue by alvaropinot** [educastellano/jsonlint-tree#4]
I think It could be really useful if an exit code is added so this can be used in CI environments :)

something like exit 0 for no errors and exit 1 whenever there are some failures. This way the this could be integrated into the npm scripts of a project for example.

I did this quick hack as a poc, if you like the idea I'll be glad to submit a PR :)

```js
...
if (passed+failed === files.length) {
   ...
   process.exit(failed)
}
...
```

What do you think?

**Comment by alvaropinot** [comment]
there is the PR just in case :P

**Comment by educastellano** [comment]
Great! I think I'm gonna use it 😃 Thanks

**Comment by educastellano** [comment]
released @1.2.0 👍
